### DISCUSSION TITLE
Embed Uiuapad?

### DISCUSSION THREAD

**Discussion by memark** [uiua-lang/uiua#243] [General]
Is there a way to embed a Uiuapad on another web page? I'm currently writing a blog post with some Uiua code, and it would be great to embed the Uiuapad with all its color highlightning and ability to run the code.

**Comment by kaikalii** [comment]
I don't think it's possible at the moment.
I'll have to look into whether it's possible to make it available through an iframe.

**Comment by kaikalii** [comment]
You can now embed the editor in an `<iframe>`.
Just replace the `pad` in a pad link with `embed` or `embedpad`.
[Here](https://www.uiua.org/embed?src=0_2_0__4oavM1804oehMTIK) is an example.

**Comment by memark** [comment]
Thank you for that quick work!
